### PR TITLE
fix cluster mismatch in envoyfilters and add arch suffix for waypoint image

### PR DESCRIPTION
**Pull request by YaoZengzeng**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ Note: * Planning
     default   istio-waypoint   10.96.143.232   True         5m7s
     ```
 
-    Add annotation "sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint:v0.3.0" to the `default` gateway. Then gateway pod will
+    Add annotation "sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint-{arch}:v0.3.0" to the `default` gateway, convert `{arch}` to the architecture of the host, such as `x86`. Then gateway pod will
     restart. Now kmesh is L7 enabled!
```
Comment (LiZhenCheng9527) [inline README.md:151]: Is there an extra line break?

File: README-zh.md [hunk 2]
```diff
@@ -148,7 +148,7 @@ Kmesh创新性的提出将流量治理下沉OS，在数据路径上无需经过
     default   istio-waypoint   10.96.143.232   True         5m7s
     ```
 
-    在`default` gateway的annotations当中添加`sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint:v0.3.0`。在gateway pod重启之后，kmesh就具备L7能力了！
+    在`default` gateway的annotations当中添加`sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint-{arch}:v0.3.0`，将`{arch}`转换为所在宿主机的架构，例如`x86`。在gateway pod重启之后，kmesh就具备L7能力了！
```
Comment (LiZhenCheng9527) [inline README-zh.md:151]:
```suggestion
    在`default` gateway的annotations当中添加`sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint-{arch}:v0.3.0`，将`{arch}`转换为所在宿主机的架构，例如`x86`。现在拥有x86和arm的waypoint镜像。在gateway pod重启之后，kmesh就具备L7能力了！

```

File: README.md [hunk 3]
```diff
@@ -147,8 +147,7 @@ Note: * Planning
     default   istio-waypoint   10.96.143.232   True         5m7s
     ```
 
-    Add annotation "sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint:v0.3.0" to the `default` gateway. Then gateway pod will
-    restart. Now kmesh is L7 enabled!
+    Add annotation "sidecar.istio.io/proxyImage: ghcr.io/kmesh-net/waypoint-{arch}:v0.3.0" to the `default` gateway, convert `{arch}` to the architecture of the host, such as `x86`. Then gateway pod will restart. Now kmesh is L7 enabled!
```
Comment (LiZhenCheng9527) [inline README.md:150]: ditto

**Review by LiZhenCheng9527 — COMMENTED** [review]
It's better move the Readme section on waypoint to the quickstart document. 
Post a link here.

File: README.md (hunk 3 above)
Comment (YaoZengzeng) [reply README.md]: improved

**Comment by YaoZengzeng** [comment]
> It's better move the Readme section on waypoint to the quickstart document. Post a link here.

will do :)

**Comment by YaoZengzeng** [comment]
@hzxuzhonghu PTAL

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**